Client/Client.Qt/Widgets/PopupWidget.cpp

```diff
@@ -31,6 +31,16 @@ void PopupWidget::hideEvent(QHideEvent*)
     if (_deleteOnHide) deleteLater();
 }
 
+void PopupWidget::setMessage(QString text) 
+{ 
+    if (!_messageText)
+    {
+        _messageText = std::make_unique<QLabel>(text, this);
```

Comment:
In case of uninitialized __messageText_ you are setting the _text_ value two times.